.github/workflows/lint.yml

```diff
@@ -1,10 +1,11 @@
 name: CI
 
 on:
+  push:
+    branches: [master]
   pull_request:
     paths:
       - "**.js"
-      - "**eslint**"
```

Comment:
Why do you removing this? It supposes to run lint what ESlint configuration was changed

Reply:
@tinovyatkin Because there are no more eslint configuration files.  Everything appears to have been moved to xo and package.json

Reply:
Ok, but how it hurts to keep this line? If some contributor will (for some reason) add `.eslintignore` or `.eslintrc` we would like this workflow to run, don't we?

Reply:
@tinovyatkin Good point. I've made this change